ci/travis/run.sh

```diff
@@ -26,11 +26,25 @@ else
       echo "local_repodata_ttl: 1800" >> ~/.condarc
       mkdir -p ~/.conda
       conda create -n blarg1 -yq python=2.7
-      conda create -n blarg3 -yq python=3.5
+      conda create -n blarg3 -yq python=3.6
       conda create -n blarg4 -yq python nomkl numpy pandas svn
-      /opt/conda/bin/py.test -v -n 2 --basetemp /tmp/cb --cov conda_build --cov-append --cov-report xml -m "not serial" tests --forked
-      # install conda-verify from its master branch, at least for a while until it's more stable
-      pip install git+https://github.com/conda/conda-verify.git
-      /opt/conda/bin/py.test -v -n 0 --basetemp /tmp/cb --cov conda_build --cov-report xml -m "serial" tests
+      
+      SLOW_MARK="and not slow"
+      if [[ $SLOW_TESTS == "true" ]]; then
+          SLOW_MARK="and slow"
+      fi
+
+      if [[ "$SANITY" == "true" ]]; then
+          pip install git+https://github.com/conda/conda-verify.git
+          /opt/conda/bin/py.test -v -n auto --basetemp /tmp/cb --cov conda_build --cov-append --cov-report xml -m "sanity and not slow and not serial" tests
+          /opt/conda/bin/py.test -v -n 0 --basetemp /tmp/cb_serial --cov conda_build --cov-append --cov-report xml -m "sanity and not slow and serial" tests
+      else
+          /opt/conda/bin/py.test -v -n auto --basetemp /tmp/cb --cov conda_build --cov-append --cov-report xml -m "not serial $SLOW_MARK and not sanity" tests
+          # install conda-verify from its master branch, at least for a while until it's more stable
+          pip install git+https://github.com/conda/conda-verify.git
+          if [[ $SLOW_MARK == "and not slow" ]]; then
```

Comment:
This is a workaround because ``pytest`` will exit with code ``5``if no tests are selected.
As there is no test with markers ``slow`` and ``serial` this script will return exit code ``5`` and it will fail on CI